sig-architecture/annual-report-2021.md

```diff
@@ -0,0 +1,102 @@
+ - [December, 2020](https://youtu.be/rnNqcUeCD8E) at KubeCon NA, published on YouTube. We do not have anything lined up for Kubecon EU. We will prioritize giving our next update in Kubecon NA.
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+- All SIG chairs are active and attend most bi-weekly full SIG meetings. We do not have separate tech leads.
+
+- For the conformance subproject, all owners are active in PR review and meeting attendance.
+
+- For the production readiness subproject, all owners are active in both KEP reviews and meetings.
+
+- For the enhancements subproject, all owners are all active.
+
+- For the API review subproject. There is no meetings as such. The subproject coordinates via a project board.
+
+- For the code organization subproject, @dims and @liggitt run regular meetings and review PRs and issues, pulling in other owners as needed.
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+- Mailing list members, though based on bounces during calendar updates, there are quite a few out-of-date memberships.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+- PRR assigns specific approvers to each KEP that is targeting a release. Therefore, we have a clear picture of how much each
+  approver must do each cycle. In 1.21, each approver reviewed 22 KEPs. for the 1.22 cycle, we have a new approver (@ehashman) to
+  help reduce the load on each approver. Go Elana!
+- Code Organization is coming along. Though we haven't updated dep-approvers in a while. Currently the root OWNERS are a bottleneck as their time is limited for dependency updates. We have a new approval plugin that wiiiii
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+- SIG Architecture owns very little code. The API review and Production Readiness Review projects do have reviewers and approvers
+  for those specialized purposes. Each of those programs has a documented shadow program to add new reviewers and approvers.
+- The main meeting itself is losing steam a bit. We could use more across the board attendance. We have to find ways to do this with better timely topics.
+- The sub projects are in sustainable shape for the most part. We could use more people in conformance, code-organization, and PRR for sure.
+- We want to rework the API reviews as well to make sure SIGs can for the most part do their own API reviews.
+- The Enhancements subproject has some tooling code, and is a good place for newer folks to contribute. It has been quite active recently, and as a process-focused subproject is a place where program managers can be 
```

Comment:
can be...?

Also I'd add a bulletpoint here about getting new members via the RT Enhancements team, who have on the ground experience with the KEP lifecycle to keep the sub project in touch with the on the ground experience (or something???).

Reply:
..."a place where program managers can be..."—could finish this sentence by stating something like:

"is a place where program managers can collaborate with other contributors on optimising processes regarding feature development, deprecations, policy rollouts, and other substantial changes."